python/additions.py

```diff
@@ -10,14 +10,12 @@
 # calculate the discriminant
 d = (b**2) - (4*a*c)
 
-# find two solutions
+# find both solutions
```

Comment:
This is one comment
that isn't that short.

Maybe try another one right after